src/Common.hpp

```diff
@@ -51,6 +52,13 @@ bool ConvertToLongPath(std::wstring* path);
 
 void InitCommon(void);
 
+//-----------------------------------------------------------------------------
+// File handling
+//-----------------------------------------------------------------------------
+
+// Make sure UTF-8 filenames are open and created correctly (specifically on Windows with multibyte character streams)
+FILE* OpenFile(const char* filename, const char* mode);
+
```

Comment:
I would merge this into the "Directories" section, since this function (you could rename "Directories" to "Filesystem").